src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  @include card-margin-right($right);
```

Comment:
It's not BEM mix
check how to do it correctly [BEM mix](https://en.bem.info/methodology/key-concepts/#mix)